app/styles/artisans/levelTasksView.sass

```diff
@@ -0,0 +1,12 @@
+#level-tasks-view
+  #levelTable
+    width: 100%
+
+  .tasksTable
+    width: 100%
```

Comment:
Tables should have `table` CSS class (from [Bootstrap](http://getbootstrap.com/css/#tables)), which will make them 100% width.